datacenter/ucp/2.1/guides/admin/configure/content-trust/admin_tasks.md

```diff
@@ -0,0 +1,387 @@
+---
+title: Configure the UCP and DTR servers for content trust
+description: Configuration tasks for using content trust on UCP and DTR servers
+---
+
+These tasks allow an administrator to set up the UCP and DTR servers to require
+content trust and to delegate the ability to sign images to UCP users. For an
+overview of content trust in UCP, see [Run only the images you trust](index.md).
+
+After completing these steps, continue to
+[Client configuration for content trust in UCP](client_configuration.md).
+
+## Prerequisites
+
+Before completing these tasks, set up the teams for signing and add users to
+them. For instance, if your business requirement is that images need to be
+signed by `engineering`, `security`, and `quality` teams, set up those teams and
+add the appropriate users to them. See
+[Set up teams](/datacenter/ucp/2.1/guides/admin/manage-users/create-and-manage-teams.md)
+and
+[Create and manage users](/ucp/2.1/guides/admin/manage-users/create-and-manage-users.md).
+
+## Overview
+
+The administrator needs to complete the following tasks to configure UCP and DTR
+for image signing.
+
+2.  [Configure UCP](#configure-ucp) to only allow signed images to be used.
+
+3.  [Set up the Docker Notary CLI client](#set-up-the-docker-notary-client)
+    locally so that the administrator can initialize the trusted image
+    repository.
+
+4.  [Initialize the trusted image repository](#initialize-the-trusted-image-repository)
+    which will store trusted images.
+
+5.  [Delegate image signing](#delegate-image-signing) so that the appropriate
+    users are able to sign images. This step is optional, and assumes that the
+    UCP administrator will not be the only one signing images.
+
+## Configure UCP
+
+This step configures UCP to only allow deployment of signed images, as well as
+the teams that must sign an image before it can be trusted. Set up the users
+and groups before starting this step.
+
+1.  Go to the **UCP web UI**, navigate to the **Admin Settings** page, and click
+    the **Content Trust** menu item.
+
+    ![Content Trust settings](/datacenter/images/ucp_content_trust_settings.png)
+
+2.  Select the **Only run signed images** option.
+
+    Click the **REQUIRE SIGNATURE FROM ALL OF THESE TEAMS** field and choose
+    one or more teams. DDC will consider an image to be trustworthy only if it
+    is signed by a member of _every team_ you select. The requirement can be
+    fulfilled by a single user who is a member of all the teams, or by a signer
+    in each team.
+
+    > **Note**: If you don't specify any team (by leaving the field blank), an
+    > image will be trusted as long as it is signed by any UCP user whose keys
+    > are [configured in the Notary client](#set-up-the-docker-notary-client).
```

Comment:
Non-blocking: I think this may make more sense as just "any UCP user", because it sounds like in order to be trusted, they have to be a UCP user *and* also something else.  I get what you are saying, though - in order for them to sign at all, their keys have to first be configured to sign in notary, so effectively that is true.  

It mostly seems confusing because the contrast with the above (where configuring keys is not mentioned at all) - like if you don't specify teams, you have to configure the keys.  But if you specify teams, you do not.